README.md

```diff
@@ -448,40 +448,42 @@ projects/
+| extension                                | _/api/config/v1/extensions_                                                 | `Read Configuration` & `Write Configuration`                                                                        |
+| kubernetes-credentials                   | _/api/config/v1/kubernetes/credentials_                                     | `Read Configuration` & `Write Configuration`                                                                        |
+| maintenance-window                       | _/api/config/v1/maintenanceWindows_                                         | `Deprecated: Configure maintenance windows`                                                                         |
+| management-zone                          | _/api/config/v1/managementZones_                                            | `Read Configuration` & `Write Configuration`                                                                        |
+| notification                             | _/api/config/v1/notifications_                                              | `Read Configuration` & `Write Configuration`                                                                        |
+| reports                                  | _api/config/v1/reports_                                                     | `Read Configuration` & `Write Configuration`                                                                        |
+| request-attributes                       | _/api/config/v1/service/requestAttributes_                                  | `Read Configuration` & `Capture request data`                                                                       |
+| request-naming-service                   | _/api/config/v1/service/requestNaming_                                      | `Read Configuration` & `Write Configuration`                                                                        |
+| slo                                      | _/api/v2/slo_                                                               | `Read SLO` & `Write SLOs`                                                                                           |
+| synthetic-location                       | _/api/v1/synthetic/locations_                                               | `Access problem and event feed, metrics, and topology` & `Create and read synthetic monitors, locations, and nodes` |
+| synthetic-monitor                        | _/api/v1/synthetic/monitors_                                                | `Create and read synthetic monitors, locations, and nodes`                                                          |
+| failure-detection-parametersets          | _/api/config/v1/service/failureDetection/parameterSelection/parameterSets_  | `Read Configuration` & `Write Configuration`                                                                        |
```

Comment:
The table was sorted alphabetically. Please move the new lines to the correct place. :pray:

Reply:
Ok, will do